### ISSUE TITLE
Failures on [ VM ] Send 'dart --help [--verbose]' output to stdout

### ISSUE THREAD

**Issue by a-siva** [dart-lang/sdk#46845]
There are new test failures on [[ VM ] Send 'dart --help [--verbose]' output to stdout](https://github.com/dart-lang/sdk/commit/cccc0293c540ebee7116d98ef6cb24c1caf9512e).

The tests
```
co19_2/LibTest/io/Process/runSync_A01_t01 RuntimeError (expected Pass)
co19_2/LibTest/io/Process/runSync_A02_t01 RuntimeError (expected Pass)
co19_2/LibTest/io/Process/runSync_A02_t02 RuntimeError (expected Pass)
co19_2/LibTest/io/Process/run_A01_t01 RuntimeError (expected Pass)
co19_2/LibTest/io/Process/run_A02_t01 RuntimeError (expected Pass)
co19_2/LibTest/io/Process/run_A02_t02 RuntimeError (expected Pass)
co19_2/LibTest/io/Process/start_A04_t01 RuntimeError (expected Pass)
co19_2/LibTest/io/Process/start_A06_t01 RuntimeError (expected Pass)
```
are failing on configurations
```
dartk-win-release-x64
```

**Comment by bkonyi** [comment]
We need to file an issue against co19 to get these fixed as we've changed where `--version` prints to.

**Comment by eernstg** [comment]
Closing: @bkonyi, I was just getting an overview of co19 related issues and came across this one. It looks like

> We need to file an issue

was resolved when dart-lang/co19#1134 was created (and it's now closed). Please reopen if I got it wrong.
